src/Config/LoggerFilters.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Config
+{
+    public class LoggerFilters
```

Comment:
If we make this a static class, the ValidFilters property might get loaded at initialization time of any project that depends on `Core` (if we move it to the `Core` project), without us needing to call the `AddFilter` function. Can you please try that?